apps/web/src/libs/algorand-adapter.ts

```diff
@@ -0,0 +1,125 @@
+import type {
+  Algodv2,
+  makePaymentTxnWithSuggestedParamsFromObject,
+  Transaction,
+} from 'algosdk'
+
+import { EventEmitter } from './event-emitter'
+
+const algosdkLoader = import('algosdk')
+
+export enum ChainType {
```

Comment:
Do we want to put some of these interfaces in schemas? Perhaps there could be some cross over with the AlgorandAdapter in the API?

Reply:
Potentially. For now let's keep them separate as they have different priorities.